src/test/java/uk/gov/pay/ledger/transaction/dao/TransactionDaoIT.java

```diff
@@ -280,4 +285,10 @@ public void findTransactionByParentIdAndGatewayAccountId_shouldFilterByParentExt
         assertThat(transactionEntity.getId(), is(transactionWithParentExternalId.getId()));
 
     }
+
+    @Test
+    public void sourceTypeInDatabase_shouldMatchValuesInEnum() {
+        var sourceArray = Arrays.stream(Source.values()).map(Enum::toString).collect(Collectors.toList());
+        transactionDao.getSourceTypeValues().forEach(x -> assertThat(sourceArray.contains(x), is(true)));
```

Comment:
Should this assertion be other way around ? i.e., for all Source enum constants there should be an entry in DB

If a new enum constant say `CARD_IVR` is added to enum but not to DB, then inserting a transaction record with source CARD_IVR will result in Postgres exception